.github/workflows/package.yml

```diff
@@ -0,0 +1,112 @@
+# .github/workflows/build-wheels.yml
+name: Build and Package Wheels
+
+on:
+  pull_request:
+  push:
+
+env:
+  LIBRDKAFKA_VERSION: v2.6.1-gr
+
+jobs:
+
+  build-linux-x64:
+    name: Build wheels for Linux x64
+    runs-on: ubuntu-20.04
+    env:
+      OS_NAME: linux
+      ARCH: x64
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - uses: actions/checkout@v4
+      - name: Build wheels
+        run: |
+          ./tools/wheels/build-wheels.sh "${LIBRDKAFKA_VERSION#v}" wheelhouse
+          tar -czf wheelhouse-linux-${ARCH}.tgz wheelhouse
+          ls -al
```

Comment:
what's the purpose of these commands?

Reply:
It was a debug command, thanks for catching it.